Core/Helper.php

```diff
@@ -90,4 +90,39 @@ public static function getFloatFromString(string $sNumber): float
     {
         return (float) preg_replace('/\.(?=.*\.)/', '', str_replace(',', '.', $sNumber));
     }
+
+    /**
+     * Validates the string for e-mail address format
+     *
+     * @param string $sEmail
+     * @return bool
+     */
+    public static function isEmailValid($sEmail)
+    {
+        return !!filter_var($sEmail, FILTER_VALIDATE_EMAIL);
+    }
+
+    /**
+     * Returns list of modules
+     *
+     * @return array;
+     */
+    public static function getModulesList()
+    {
+        $sModulesDir = Registry::getConfig()->getModulesDir();
+        $oModuleList = oxNew(\OxidEsales\Eshop\Core\Module\ModuleList::class);
+        $aModules = $oModuleList->getModulesFromDir($sModulesDir);
+        return $aModules;
+    }
+
+    /**
+     * Check if $sModuleId is this plugin id
+     *
+     * @param string $sModuleId
+     * @return bool
+     */
+    public static function isThisModule($sModuleId)
```

Comment:
I don't think this should be in the Helpers. A const in `ModuleSupport` should be enough.

Reply:
There is helper method returning list of modules. Why not having this check also in helper?